packages/synapse-react-client/src/style/components/_user-card-list-groups.scss

```diff
@@ -38,6 +35,8 @@
           font-weight: bold;
           flex-grow: 1;
           flex-shrink: 1;
+          text-transform: capitalize;
+          box-sizing: border-box;
```

Comment:
Add back styling to the study nav buttons previously provided by bootstrap-4-backport in UserCardListGroups